src/hooks/useScene.ts

```diff
@@ -3,14 +3,15 @@ import * as THREE from 'three';
 
 import useFetchPanoImage from './useFetchPanoImage';
 
+const scene = new THREE.Scene();
```

Comment:
Had to pull out scene and camera out of useScene function for now, since if component gets updated useScene generates a new pair of scene and a camera. That causes screen to go black. Try putting this logic back to useScene to see what I mean.